### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ryanpetersonOF**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,41 +6,42 @@ OpenFin Layouts delivers window management and layout user experience across the
...
-   - On inclusion of the client API, windows and groups can also be undocked programmatically.
+   - Windows can be undocked by pressing `CTRL+SHIFT+U` or `CMD+SHIFT+U` when the window has focus
+   - On inclusion of the client API, windows and groups can also be undocked programmatically
 - Tabbing
-   - Dropping a window on-top of another window will create tabbed windows with a tabstrip on top
-   - Tabs can be reorder and renamed.
-   - Minimize / maximize / restore / close on the tabstrip affects the whole tabgroup (tabstrip and tabbed windows)
+   - Dropping a window on-top of another window will tab the windows together
+   - Dropping a window on-top of an existing tab group will add it to that group
+   - Tabs can be reorder and renamed
```
Comment (bryangaleOF) [inline README.md]: reordered

File: README.md [hunk 2]
```diff
@@ -6,41 +6,42 @@ OpenFin Layouts delivers window management and layout user experience across the
...
+   - Minimize / maximize / restore / close on the tabstrip affects the whole tab group (tabstrip and tabbed windows)
    - Application developers can provide their own tabstrip - simply by hosting a templated html, initiated on application startup
-   - The service comes with a win10 like default tabstrip that will used if no custom tabstrip is defined
-- Save and restore
-   - The service has APIs for getting and restoring layouts
+   - The service comes with a Chrome like default tabstrip that will used if no custom tabstrip is defined
```
Comment (bryangaleOF) [inline README.md]: 'Chrome-like'
'that will be used'

File: README.md [hunk 3]
```diff
@@ -6,41 +6,42 @@ OpenFin Layouts delivers window management and layout user experience across the
...
+- Save & Restore
+   - The service has APIs for generating and restoring workspaces
    - The provided demo showcases how a layout manager application could use the APIs to manage layouts
 - APIs
-   - API available to undock, ungroup, tab / un-tab , save / restore a layout or opt-out of service functionality.
+   - APIs available to undock, ungroup, tab / un-tab, save / restore a layout or opt-out of service functionality.
```
Comment (bryangaleOF) [inline README.md]: 'and save / restore'?

File: README.md [hunk 4]
```diff
@@ -6,41 +6,42 @@ OpenFin Layouts delivers window management and layout user experience across the
...
 - Hosting
    - The latest production version OpenFin Layouts will by default be served from OpenFin's CDN
-   - For testing / dev purposes, a customers can specify an absolute version/location of a the service by providing the full URL in the services section of the app manifest (see 'Manifest declaration' below)
+   - For testing / dev purposes, customers can specify an absolute version/location of the service by providing the full URL in the services section of the app manifest (see 'Manifest declaration' below)
```
Comment (bryangaleOF) [inline README.md]: Consistency of putting spaces either side of /

File: README.md [hunk 5]
```diff
@@ -6,41 +6,42 @@ OpenFin Layouts delivers window management and layout user experience across the
...
    - To self-host versions of the service, each release is also deployed to the CDN as a zip file, available at `https://cdn.openfin.co/services/openfin/layouts/<version>/layouts-service.zip`
 
 ## Getting Started
 
-Integrating the Layouts Service within an application is done in two steps, add the service to application manifest and import the API:
+Integrating the Layouts Service within an application is done in two steps.  Add the service to the application manifest and import the API:
```
Comment (bryangaleOF) [inline README.md]: Would put 'application manifest, and import the API' to make it clear you're enumerating the steps

File: README.md [hunk 6]
```diff
@@ -6,41 +6,42 @@ OpenFin Layouts delivers window management and layout user experience across the
 
 This project consist of 3 parts:
 1. The Layouts Service, which manages the state of windows, tabs and layouts
-2. The Layouts Client, which exposes calls to programmatically control snap/dock, tabbing and save/restore.
+2. The Layouts Client, which exposes calls to programmatically control snap/dock, tabbing and save/restore
```
Comment (bryangaleOF) [inline README.md]: I think in most places in the readme we're putting one space of padding around '/'. Would prefer we didn't but more bothered about consistency than which we do

File: README.md (hunk 3 above)
Comment (bryangaleOF) [reply README.md]: Hmm, actually could stay as is, but might read better if the 'or' is changed to an 'and'

File: README.md (hunk 6 above)
Comment (ryanpetersonOF) [reply README.md]: I agree.  I have removed all spacing throughout

File: README.md [hunk 7]
```diff
@@ -6,41 +6,42 @@ OpenFin Layouts delivers window management and layout user experience across the
...
 - Hosting
    - The latest production version OpenFin Layouts will by default be served from OpenFin's CDN
-   - For testing / dev purposes, a customers can specify an absolute version/location of a the service by providing the full URL in the services section of the app manifest (see 'Manifest declaration' below)
+   - For testing/dev purposes, customers can specify an absolute version/location of the service by providing the full URL in the services section of the app manifest (see 'Manifest declaration' below)
    - To self-host versions of the service, each release is also deployed to the CDN as a zip file, available at `https://cdn.openfin.co/services/openfin/layouts/<version>/layouts-service.zip`
 
 ## Getting Started
 
-Integrating the Layouts Service within an application is done in two steps, add the service to application manifest and import the API:
+Integrating the Layouts Service within an application is done in two steps.  Add the service to the application manifest, and import the API:
```
Comment (bryangaleOF) [inline README.md]: Double spacing after the full stop

Comment (ryanpetersonOF) [reply README.md]: Fixed±

**Pull request merged**